packages/i18nify-js/README.md

```diff
@@ -68,6 +68,30 @@ resetState();
 
 This module's your go-to guru for everything currency/number-related. 🤑 It's all about formatting, validations, and handy tricks to make dealing with money/numbers a breeze. Here are the cool APIs and utilities this Currency Module gives you to play with! 🚀💸
 
+#### convertFromLowerDenominationToUpperDenomination(amount, currencyCode)
```

Comment:
Keep the 2nd parameter as `options` . Keep the contract similar to what's there for other APIs.